prow/tide/status.go

```diff
@@ -56,12 +68,13 @@ type statusController struct {
 	// lastSyncStart is used to ensure that the status update period is at least
 	// the minimum status update period.
 	lastSyncStart time.Time
-	// latestPR is the time of the most recent result
-	latestPR      time.Time
-	previousQuery string
 
 	sync.Mutex
 	poolPRs map[string]PullRequest
+
+	storedState
+	opener io.Opener
```

Comment:
this could be named in a more descriptive way

Reply:
I refactored this to open whatever file we want, using the specified remote credentials. So this isn't a "storedStateOpener", and therefore I think this name is reasonable